doc/nrf/ug_matter_configuring.rst

```diff
@@ -0,0 +1,59 @@
+.. _ug_matter_configuring:
+
+Configuring Matter
+##################
+
+When you program any of the available Matter samples to the supported development kits, you automatically build the Matter stack for the nRF Connect platform.
+The development kit and the application running Matter stack that is programmed on the development kit together form the Matter accessory device.
+
+For the |NCS|, the default use case of a Matter accessory device is within a Matter network that is built on top of a low-power, 802.15.4-compatible Thread network.
```

Comment:
I think we could make this stronger as it's not default use case - it's the only use case :) Something like "Currently, NCS only supports Matter stack built on top of a low-power..." would be more accurate.

Reply:
Rephrased.